### PR TITLE
💻 Refactor and simplify EventViewToggle.tsx

### PR DESCRIPTION
**Pull request by evtyz**
### Before:
desktop
tablet
phone

### After:
desktopafter
tabletafter
phoneafter

## Purpose
Refactor code for readability and performance. Objective: preserve functions and visuals while conforming better to front-end standards.

## Approach
Remove redundant code and re-implement it in a more concise and readable way. Example: removing nested containers, replacing spacing components with CSS.

## Testing
Visual inspection on before & after screenshots. Responsiveness on three different viewport sizes tested as well.

https://github.com/loolabs/waterpark/issues/115

### CODE REVIEW

**Review by wang-em — DISMISSED** [review]
LGTM!

File: client/components/events/EventViewToggle.tsx [hunk 1]
```diff
@@ -79,8 +70,6 @@ export const EventViewToggle = ({ events }: EventViewProps) => {
 
   return (
     <EventViewToggleContainer>
-      <EventViewToggleButtonsContainer>
-        <EventViewToggleButtonsSpacer />
         <EventViewToggleButtons>
```
Comment (xujustinj) [inline client/components/events/EventViewToggle.tsx:73]: indentation is kind of broken here, can we re-run Prettier on this?

**Review by xujustinj — APPROVED** [review]
non-blocking formatting issue, approved anyway

File: client/components/events/EventViewToggle.tsx (hunk 1 above)
Comment (evtyz) [reply client/components/events/EventViewToggle.tsx]: Done

**Review by xujustinj — APPROVED** [review]

**Review by wang-em — APPROVED** [review]

**Pull request merged**